libs/@guardian/react-crossword/src/components/Clues.tsx

```diff
@@ -13,12 +10,19 @@ const title = css`
 
 type Props = {
 	direction: Direction;
-	entries: CrosswordEntry[];
+	entries: CAPIEntry[];
```

Comment:
this feels like a strange change to need to make - maybe we need to check the types

Reply:
That function could return a Map<EntryId, CrosswordEntry>  and then Array.from(entries.values())...